dangerzone/util.py

```diff
@@ -28,21 +27,21 @@ def get_resource_path(filename: str) -> str:
     if getattr(sys, "dangerzone_dev", False):
         # Look for resources directory relative to python file
         project_root = pathlib.Path(__file__).parent.parent
-        prefix = project_root.joinpath("share")
```

Comment:
This might be a question of preference, and I don't have any strong opinion on the matter. This is in a separate commit, so can be reverted if you prefer the version with `.joinpath()`.